### PR TITLE
chore: add schema.json for es-cluster

### PR DESCRIPTION
**Pull request by Yuyinzi**
resolve [kubeclocks/issues/6878](https://github.com/apecloud/kubeblocks/issues/6878)

### CODE REVIEW

File: addons/elasticsearch-cluster/values.schema.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      "description": "The override fullname.",
+      "type": "string",
+      "default": ""
+    },
+    "resources": {
+      "title": "resources",
+      "description": "If you do want to specify resources, uncomment the following lines after 'resources:' in values.yaml",
+      "type": "object"
+    },
+    "nodeSelector": {
```
Comment (shanshanying) [inline addons/elasticsearch-cluster/values.schema.json:57]: dito.

File: addons/elasticsearch-cluster/values.schema.json [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+      "title": "NodeSelector",
+      "type": "object"
+    },
+    "tolerations": {
+      "title": "Tolerations",
+      "type": "array",
+      "items": {}
+    },
+    "affinity": {
```
Comment (shanshanying) [inline addons/elasticsearch-cluster/values.schema.json:66]: dito.

File: addons/elasticsearch-cluster/values.schema.json [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+          "default": true
+        },
+        "data": {
+          "type": "object",
+          "properties": {
+            "storageClassName": {
+              "type": ["string", "null"],
+              "default": null
+            },
+            "size": {
```
Comment (shanshanying) [inline addons/elasticsearch-cluster/values.schema.json:104]: `size` cannot be an arbitary string. You may refer to 
https://github.com/apecloud/kubeblocks-addons/blob/676f181686f5018a19608c51f887b554cbbc9f00/addons/apecloud-mysql-cluster/values.schema.json#L29-L30

Comment (Yuyinzi) [reply addons/elasticsearch-cluster/values.schema.json:151]: Thanks for reviewing! I've added limitations for both replicas&resources&persistence

**Comment by Yuyinzi** [comment]
Any suggestions then? Thanks! @shanshanying

**Review by free6om — APPROVED** [review]

**Pull request merged**